src/test/java/teammates/ui/webapi/UpdateStudentProfileActionTest.java

```diff
@@ -215,5 +200,18 @@ private StudentProfileUpdateRequest createInvalidUpdateRequestForProfileWithScri
     protected void testAccessControl() {
         verifyInaccessibleWithoutLogin();
         verifyInaccessibleForUnregisteredUsers();
+        testActionForbidden();
+    }
+
+    private void testActionForbidden() {
```

Comment:
very minor nit: perhaps can rename this function to `testActionUnauthorized` and also the test case name to `______TS("Unauthorized Access Case: updating another student's profile");`

Reply:
Strictly speaking, unauthorized is for when there is no login credentials whereas forbidden is for when there is wrong login credentials (think 401 vs 403, although we treat them the same in the API layer). This test case is closer to 403.

Reply:
Ohh I see what you mean